app/src/main/java/com/kickstarter/viewmodels/DiscoveryViewModel.java

```diff
@@ -214,7 +214,7 @@ public DiscoveryViewModel(final @NonNull Environment environment) {
       .take(1)
       .map(Intent::getAction)
       .filter(Intent.ACTION_MAIN::equals)
-      .map(__ -> DiscoveryParams.builder().staffPicks(true).build());
+      .map(__ -> DiscoveryParams.builder().build());
```

Comment:
Are we sure we want to alter the functionality here? Seems like this is a behavioral change rather than just language?

I think it might also introduce a bug with the onboarding – https://github.com/kickstarter/android-oss/pull/31/files#diff-35cad2327f6b5b0f53fd5b9f64ee033bR278 onboarding is only visible when staff picks are true, but now we've turned them off by default?